starlight/en.py

```diff
@@ -61,6 +61,10 @@ def gap_date_range(a):
     17: """Perfect notes will restore <span class="let">{0}</span> health""",
     18: """Great/Perfect notes will restore <span class="let">{0}</span> health""", #provisional
     19: """Nice/Great/Perfect notes will restore <span class="let">{0}</span> health""", #provisional
+    20: """your currently activating skills will receive a boost effect""", # depending on skill type? provisional
```

Comment:
Is the boost applied to all skills that activate while Skill Boost is up, or just the skills that are up when Skill Boost procs?